### PR TITLE
Make compatible with Laravel 8

### PR DESCRIPTION
**Pull request by 99kennetn**
This pr makes the repository compatible with laravel 8
No changes had to be made other than version compatibilities.

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -11,13 +11,13 @@
         }
     ],
     "require": {
-        "php": ">=7.0",
+        "php": ">=7.3",
         "beberlei/assert": "^2.7.6",
-        "guzzlehttp/guzzle": "^6.2",
-        "symfony/options-resolver": "^3.0 || ^4.0"
+        "guzzlehttp/guzzle": "^7.0.1",
```
Comment (loevgaard) [inline composer.json]:
```suggestion
        "guzzlehttp/guzzle": "^6.2 || ^7.0.1",
```

File: composer.json [hunk 2]
```diff
@@ -11,13 +11,13 @@
...
+        "symfony/options-resolver": "^5.1"
```
Comment (loevgaard) [inline composer.json]:
```suggestion
        "symfony/options-resolver": "^3.4 || ^4.4 || ^5.0"
```

**Comment by loevgaard** [comment]
Do you use the library as `dev-master`?

**Comment by loevgaard** [comment]
Because I can see that I haven't made a release in quite a while and there are lot of commits pending

**Comment by 99kennetn** [comment]
until now we have been using dev-master, but if at all possible a new version nr would be nice.

**Comment by loevgaard** [comment]
I will just release 3.0

**Comment by loevgaard** [comment]
Done :)

**Comment by 99kennetn** [comment]
Thanks, it has been download now with the new version :D thanks for the quick reponse :)

**Pull request merged**